### ISSUE TITLE
[Server] Cannot connect clients to server

### ISSUE THREAD

**Issue by Jeremy-Laier** [FTBTeam/FTB-Interactions#956]
## Bug Report:
Upon creating 2.0.0 FTB: interactions server using the provided linux runnable, once everything is running connecting gives this error.

```
Connection Lost
Internal Exception: com.google.common.util.concurrent.UncheckedExecutionException: java.lang.IllegalStateException: Recipe minecraft:shears (net.minecraft.item.crafting.ShapedRecipes@6cb59aa) produces unregistered item minecraft:shears (net.minecraft.item.ItemShears@6fb51e17)
```
## Expected Behavior/Suggestions:
Should be able to connect, not sure how to fix

## Is It Repeatable? Steps to Reproduce: 
1. Run server
2. Connect
3. Failure to join

## Link to Log or Crash File Paste**: 
Only error is above, server only lists client connecting and disconnecting

## Mod/s Affected: 
Not sure 

## Known Fix: 
None :(

**Comment by Jeremy-Laier** [comment]
Looks like this should have been fixed a while ago? https://github.com/FTBTeam/FTB-Interactions/issues/482

I was not planning on using 2.0.1, but if this issue is fixed I can bump versions

**Comment by Jeremy-Laier** [comment]
This error is fixed in 2.0.

**Comment by Saereth** [comment]
yeah it is fixed in 2.0.1+ looks like you got it sorted
